metron-platform/metron-parsers/src/test/resources/org/apache/metron/parsers/cef/cyberark.json

```diff
@@ -0,0 +1,21 @@
+{
+    "\"Other info\"": "101.198.70.93",
+    "\"Safe Name\"": "Security Vulnerability Mgmt",
+    "\"Ticket Id\"": "Needed to verify config files being pulled ",
+    "deviceAction": "Retrieve password",
+    "deviceAddress": "120.99.70.3",
```

Comment:
I suspect metron will take care of the IP parsing but it may be a good idea to add ipv6 to the test units?